flutter-intellij-community.iml

```diff
@@ -46,5 +46,12 @@
     <orderEntry type="library" exported="" name="Dart SDK" level="project" />
     <orderEntry type="library" exported="" name="Dart Packages" level="project" />
     <orderEntry type="library" name="miglayout-swing" level="project" />
+    <orderEntry type="module" module-name="intellij.android.adt.ui" />
```

Comment:
what happens with your current build if you start the project in regular intellij?